var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -41,6 +42,13 @@ class PyPytest(PythonPackage):
     version("3.0.7", sha256="b70696ebd1a5e6b627e7e3ac1365a4bc60aaf3495e843c1e70448966c5224cab")
     version("3.0.2", sha256="64d8937626dd2a4bc15ef0edd307d26636a72a3f3f9664c424d78e40efb1e339")
 
+    variant(
+        "dev",
+        description="Include optional dependencies, pluggy and tomli",
```

Comment:
I don't understand this variant. These dependencies are not optional, they are required in order to install pytest